changelog/README.md

```diff
@@ -2,6 +2,7 @@
 
 ### 3.7.0 (In progress)
 
+- [improvement] JAVA-1950: Log server side warnings returned from a query.
```

Comment:
Nit: for legacy drivers, changelog entries should be placed after the existing ones, not before.